homeassistant/components/zwave_me/sensor.py

```diff
@@ -38,28 +70,22 @@
         native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
         state_class=SensorStateClass.TOTAL_INCREASING,
     ),
+    "meterElectric_power_factor": SensorEntityDescription(
+        key="meterElectric_power_factor",
+        device_class=SensorDeviceClass.POWER_FACTOR,
+        native_unit_of_measurement=PERCENTAGE,
```

Comment:
We may need a conversion function set as an attribute in the description that is applied before returning the state. What is the native value returned by the api for this measurement? The end result state should be a percentage.

Here's an example how to apply a conversion function from a description attribute:

https://github.com/home-assistant/core/blob/78b02fe0cefe6205178516290a561678b64be2c4/homeassistant/components/airly/sensor.py#L59-L63

https://github.com/home-assistant/core/blob/78b02fe0cefe6205178516290a561678b64be2c4/homeassistant/components/airly/sensor.py#L115

https://github.com/home-assistant/core/blob/78b02fe0cefe6205178516290a561678b64be2c4/homeassistant/components/airly/sensor.py#L170

Reply:
Looks ok now